docs/changelog.rst

```diff
@@ -6,6 +6,14 @@ Development
 ===========
 - (Fill this out as you fix issues and develop your features).
 
+Changes in 0.14.1
+=================
+- Removed SemiStrictDict and started using a regular dict for `BaseDocument._data` #1630
+- Added support for the `$position` param in the `$push` operator #1566
```

Comment:
The same operator we use in drop reorder?

Reply:
`$push` yeah, but not `$position`.

> The $position modifier specifies the location in the array at which the $push operator insert elements.

Do you think it could be useful ?

Reply:
K.. $push+$position would be useful when adding to a drop.. since we currently add to the beginning of the droplets, not to the end.. but this could uses `save` anyway because of some bug with `push` and `GenericEmbeddedDocument`